src/directives/formly-form.test.js

```diff
@@ -933,12 +933,19 @@ describe('formly-form', () => {
 
       const listener = sinon.spy()
       const expression = sinon.spy()
+      const deepLinkField = getNewField()
+      deepLinkField.key = 'foo.bar'
+      deepLinkField.watcher = {
+        listener: '',
```

Comment:
Almost. We want to test the default `listener` (which is what you're modifying) so you'd omit the listener on the `deepLinkField`. Other than that, what you have is sufficient.